docs/setup/kubeedge_install.md

```diff
@@ -0,0 +1,76 @@
+# Install KubeEdge
+
+In this section, we would cover the below topics
+
+1. [Abstract](#Abstract)
+2. [Pre-Requisite](#pre-requisite)
+3. [Setup KubeEdge](#setup-kubeEdge)
+4. [Configure KubeEdge](#configure-kubeEdge)
+5. [Run KubeEdge](#run-kubeedge)
+6. [KubeEdge Pre-Check](#kubeedge-pre-check)
+## Abstract
+KubeEdge is composed of cloud and edge sides. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. So if we want to setup KubeEdge, we need to setup Kubernetes cluster, cloud side and edge side.
+
++ on `cloud side`, we need to install Docker, Kubernetes cluster and cloudcore.
++ on `edge side`, we need to install Docker, MQTT and edgecore.
+
+## Pre-Requisite
+
++ Please refer [compatibility-matrix](https://github.com/kubeedge/kubeedge#compatibility-matrix) to understand **Go dependency** and **Kubernetes compatibility** and determine what version of Docker, Kubernetes, Golang can be installed.
+
+### Cloud side (KubeEdge Master Node)
+
++ [Install golang](https://golang.org/dl/)
```

Comment:
I like to remove this here, go is only needed if you want to compile KubeEdge by yourself. I think this should be a tutorial for starters of KubeEdge and they should use the binary packages or the docker containers.